R/stations.R

```diff
@@ -193,11 +193,19 @@ stations_search <- function(name = NULL,
   }
 
   if(!is.null(coords)){
+    if(!requireNamespace("sf", quietly = TRUE)) stop("The sf package needs to be installed to search by coordinates.")
+    if(!requireNamespace("geosphere", quietly = TRUE)) stop("The geosphere package needs to be installed to search by coordinates.")
```

Comment:
Switch one of these lines to reference sp, and then keep the original code below.